roles/confluent.variables_handlers/vars/main.yml

```diff
@@ -1,11 +1,13 @@
 ---
 zookeeper_service_name: confluent-zookeeper
 zookeeper_main_package: "{{ 'confluent-server' if confluent_server_enabled|bool else 'confluent-kafka-2.12'}}"
-zookeeper_packages:
+zookeeper_community_packages:
```

Comment:
One thing to make sure of here is these packages have a very specific order: dependencies first
Why this is important is: if you run `yum install confluent-server-5.4.0` it will install confluent-common-5.4.2 as a dependency...

My concern with this implementation is if you have confluent_community set to false then the union will put confluent-metadata service after zookeeper_main_package in the list.... I can do some testing. This might be a good time for the omit filter, I need to read more on that filter though

Reply:
update: omit filter did not work but this would:
```
zookeeper_packages:
  - confluent-common
  - confluent-rest-utils
  - "{{ '' if confluent_community|bool else 'confluent-metadata-service' }}"
  - "{{ zookeeper_main_package }}"
zookeeper_final_packages: "{{zookeeper_packages | difference(['']) }}"
```

Reply:
Good catch! I will implement that

Reply:
I used your approach but tried to keep the "API" stable, by not changing the final `zookeeper_package` variable

Reply:
Looks good!